example/components/ProxyExample/ProxyExample.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
```

Comment:
Put everything into one file `ProxyExample`, do not create folder for each example.